web/frontend/src/components/slam/2d-renderer.vue

```diff
@@ -257,44 +260,41 @@ const updatePointCloud = (pointcloud: Uint8Array) => {
     }
   }
 
-  points.geometry.computeBoundingBox();
-
-  // construct grid spaced at 1 meter
-  const gridHelper = createGridHelper(points.geometry.boundingBox!);
-
-  // construct axes
-  const axesPos = createAxisHelper('AxesPos', Math.PI / 2);
-  axesPos.rotateY(Math.PI / 2);
-  const axesNeg = createAxisHelper('AxesNeg', -Math.PI / 2);
-  axesNeg.rotateY(Math.PI / 2);
-  axesNeg.rotateX(Math.PI);
-
   // add objects to scene
   scene.add(
-    gridHelper,
     points,
-    intersectionPlane,
-    axesPos,
-    axesNeg,
-    baseMarker,
-    destMarker
+    intersectionPlane
   );
 
   updateOrRemoveDestinationMarker();
 };
 
 let removeUpdate: (() => void) | undefined;
 
-onMounted(() => {
-  removeUpdate = update(() => {
-    const { zoom } = camera;
+const scaleObjects = () => {
+  const { zoom } = camera;
 
-    if (pointsMaterial) {
-      pointsMaterial.size = zoom * cameraScale;
-    }
-  });
+  if (pointsMaterial) {
+    pointsMaterial.size = zoom * cameraScale * window.devicePixelRatio;
+  }
+
+  const spriteSize = baseSpriteSize / zoom;
```

Comment:
Markers now act like HUD elements, they stay the same size regardless of zoom, which makes closely zoomed out and zoomed in views much less jarring